### PR TITLE
Complete CV Bootcamp

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -12,6 +10,108 @@
...
+        tf.keras.layers.Flatten(),
+        tf.keras.layers.Dense(128, activation='relu'),
+        tf.keras.layers.Dense(10)
+    ])
+
+    model.compile(optimizer='Adam', loss=tf.keras.losses.SparseCategoricalCrossentropy(from_logits=True), metrics=['accuracy'])
+
+    return model
+
+def plot_loss_accuracy(model_history):
```
Comment (mikashaw) [inline main.py:63]: great that you added visualizations!

File: main.py [hunk 2]
```diff
@@ -12,6 +10,108 @@
...
+
+    plt.xlabel('Epoch')
+    plt.ylabel('Accuracy')
+
+    plt.legend(['Training', 'Validation'])
+
+    plt.show()
+ 
+
+def main():
```
Comment (mikashaw) [inline main.py:103]: Great job overall - very object oriented. Comments were quite helpful and it was great that you added training loss as well as accuracy plots, just watch out for overfitting (noticed in results that validation loss went up after a few epochs). Welcome to the team :)

**Review by mikashaw — APPROVED** [review]
Great job overall - very object oriented. Comments were quite helpful and it was great that you added training loss as well as accuracy plots, just watch out for overfitting (noticed in results that validation loss went up after a few epochs). Welcome to the team :)
